pkgs/development/tools/aws-sam-cli/default.nix

```diff
@@ -4,54 +4,15 @@
 , enableTelemetry ? false
 }:
 
-let
-  py = python3.override {
-    packageOverrides = self: super: {
-      flask = super.flask.overridePythonAttrs (oldAttrs: rec {
-        version = "1.0.2";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "0j6f4a9rpfh25k1gp7azqhnni4mb4fgy50jammgjgddw1l3w0w92";
-        };
-      });
-
-      aws-sam-translator = super.aws-sam-translator.overridePythonAttrs (oldAttrs: rec {
-        version = "1.27.0";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "sha256-MlgSL0zWcbo9wUE4LdBBLBI9ufExe+2LJucin2NjRFQ=";
-        };
-      });
-
-      dateparser = super.dateparser.overridePythonAttrs (oldAttrs: rec {
-        version = "0.7.0";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "sha256-lAgoGDyTe87FMHUyEbcPZzwKmquDHkMnNImzEFON/4Y=";
-        };
-      });
-
-      cookiecutter = super.cookiecutter.overridePythonAttrs (oldAttrs: rec {
-        version = "1.6.0";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "0glsvaz8igi2wy1hsnhm9fkn6560vdvdixzvkq6dn20z3hpaa5hk";
-        };
-      });
-    };
-  };
-
-in
-
-with py.pkgs;
+with python3.pkgs;
```

Comment:
I would prefer moving this to where it is needed.

Reply:
I think this is okay, mostly because it's not mixing toolchains. Since it's a "standard" python package, I think the context is apparent enough. However, if it was also doing some c++/rust builds, and something more complicated; then I would agree that scoping is vital.

Reply:
I admit I simply replaced the with call as no more override was needed